src/app/product-details/product-details.component.html

```diff
@@ -0,0 +1,28 @@
+<mat-card  class="mat-elevation-z8">
+  <mat-card-header>Product Information</mat-card-header>
+    <mat-form-field appearance="standard">
+      <mat-label>Product Name</mat-label>
+      <input matInput id="productName" name="productName" type="text" required [(ngModel)]="product.productName">
+    </mat-form-field>
+    <br>
+    <mat-label>Category</mat-label>
+    <mat-radio-group class="detail-radio-group" [(ngModel)]="product.category">
+      <mat-radio-button class="detail-radio-button" *ngFor="let category of productCategories" [value]="category">
+        {{category}}
+      </mat-radio-button>
+    </mat-radio-group>
+    <br>
+    <mat-form-field appearance="standard">
+      <mat-label>Quantity</mat-label>
+      <input matInput id="quantity" name="quantity" type="text" required [(ngModel)]="product.quantity">
+    </mat-form-field>
+    <br>
```

Comment:
You shouldn't need to use <br> tags, you can use flex to create a column

Reply:
You can also use css grid for the same effect. Both have their pros and cons and it would be worth it to play around with both.